tests/docs/source/autoconfigurable/inherited_members.rst

```diff
@@ -0,0 +1,15 @@
+autoconfigurable - inherited-members
+====================================
+
+With the ``inherited-members`` option, we expect traits from the superclass
+TestConfigurable to show up as well as traits from TestConfigurableSubclass.
+
+.. note::
+   Several members from the ``traitlets.HasTraits`` base class are excluded for
+   now as they cause the sphinx-build warning of ``Unexpected section title`` in
+   sphinx 4.
+
+.. autoconfigurable:: test_module.TestConfigurableSubclass
+   :noindex:
+   :inherited-members:
```

Comment:
I tried to set `:inherited-members: traitlets.HasTraits`, since the [documentation](https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html) suggests this should exclude methods inherited from this class, but it doesn't seem to work.

I've tried it on `autoclass` without autodoc-traits, and `:inherited-members: parent.Class` seems to simply not do what it says.

Reply:
HMmm, what version of sphinx was used? See https://github.com/sphinx-doc/sphinx/pull/10325 added for sphinx 5.0

Reply:
5.3.0. Who knows? Not our problem, for now. It seems like it will be desirable _pretty often_ for documenting inheritors of HasTraits, which has _a lot_ of methods.